{{ cookiecutter.repo_directory }}/index.html

```diff
@@ -2,6 +2,12 @@
 <html lang="en">
 
 <head>
+    {%- if 'redirect' in cookiecutter %}
+    <meta http-equiv="refresh" content="0; {{cookiecutter.redirect.url}}">
```

Comment:
Seems like a simple change. Out of curiosity, as someone who doesn't speak HTML :), where do you go to understand syntax like this?

Reply:
Ahh .. meant to link this :wink: 
https://en.wikipedia.org/wiki/Meta_refresh